R/kpca_rbf.R

```diff
@@ -134,7 +134,7 @@ prep.step_kpca_rbf <- function(x, training, info = NULL, ...) {
       cli::cli_abort(c(
         x = "Failed with error:",
         i = as.character(kprc)
-      ))
```

Comment:
This ought to just set `parent = kprc`, I think?

Reply:
@simonpcouch do you mind showing an example? i tried for a little bit but not able to get anywhere

Reply:
we can talk more here: https://github.com/tidymodels/recipes/issues/1386